packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -76,6 +77,11 @@ const propTypes = {
    * If set to true, the AbstractModal is rendered inside a NotificationDialog.
    */
   isCalledFromNotificationDialog: PropTypes.bool,
+  /**
+   * @private
+   * If set to true, then the focus lock will get enabled.
+   */
+  enableFocusLock: PropTypes.bool,
```

Comment:
In what scenario would we want to disable the focus lock?  Should this be an option?